private/js2ts/index.mjs

```diff
@@ -0,0 +1,68 @@
+#!/usr/bin/env node
+
+
+/** This script can be used to convert plugins with ESM source into plugins with TS source. */
```

Comment:
```suggestion

/**
 * This script can be used to initiate the transition for a plugin from ESM source to
 * TS source. It will rename the files, update the imports, and add a `tsconfig.json`.
*/
```